InstallChrome.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+sudo apt install ./google-chrome-stable_current_amd64.deb
```

Comment:
- [ ] guess linux likes a new line at end. that is what this red warning is telling us. Can you press enter (thus add a newline) and push change again?

Reply:
done